specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/examples/Libraries_ListByWorkspace.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "12345678-1234-1234-1234-12345678abc",
+    "resourceGroupName": "exampleResourceGroup",
+    "workspaceName": "exampleWorkspace",
+    "ifNoneMatch": null,
+    "api-version": "2019-06-01-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "Date": "Wed, 23 Oct 2020 21:33:05 GMT",
+        "X-Content-Type-Options": "nosniff",
+        "x-ms-ratelimit-remaining-subscription-reads": "14909",
+        "x-ms-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b",
+        "x-ms-correlation-request-id": "f632df92-1140-4e19-9e19-0b1c8cbe393b"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "/subscriptions/12345678-1234-1234-1234-12345678abc/resourceGroups/exampleResourceGroup/providers/Microsoft.Synapse/workspaces/exampleWorkspaceName/libraryartifacts/exampleLibraryName1.jar",
+            "name": "exampleLibraryName1.jar",
+            "type": "Microsoft.Synapse/workspaces/libraryartifacts",
+            "properties": {
+              "name": "exampleLibraryName1.jar",
+              "path": "exampleWorkspace/libraries/exampleLibraryName1.jar",
+              "containerName": "exampleContainer",
+              "type": "jar",
```

Comment:
>type [](start = 15, length = 4)

You should call it `libraryType` to avoid confusion with `type`.

Reply:
This is library information. Wouldn't it be redundant to add library as a prefix for type. Also, in that case all attributes will be changed to have library as prefix e.g. libraryName, libraryPath which will be redundant as this information is already about library.